webapp/components/generic/SearchableInput/SearchableInput.spec.js

```diff
@@ -32,81 +32,79 @@ describe('SearchableInput.vue', () => {
   }
 
   describe('mount', () => {
-    describe('testing custom functions', () => {
-      let select
+    let select
 
-      beforeEach(() => {
-        select = wrapper.find('.ds-select')
-        select.trigger('focus')
-        select.element.value = 'abcd'
-      })
+    beforeEach(() => {
+      select = wrapper.find('.ds-select')
+      select.trigger('focus')
+      select.element.value = 'abcd'
+    })
 
-      it('opens the select dropdown when focused on', () => {
-        expect(wrapper.find('.is-open').exists()).toBe(true)
-      })
+    it('opens the dropdown when focused', () => {
+      expect(wrapper.find('.ds-select-dropdown').isVisible()).toBe(true)
+    })
 
-      it('opens the select dropdown and blurs after focused on', () => {
-        select.trigger('blur')
-        expect(wrapper.find('.is-open').exists()).toBe(false)
-      })
+    it('closes the dropdown when blurred', () => {
+      select.trigger('blur')
+      expect(wrapper.find('.ds-select-is-open').exists()).toBe(false)
+    })
```

Comment:
why couldn't the opposite of :point_up_2: be done?

Reply:
Very good question. I tried but didn't get it to work. To be honest, there are so many nested elements from the styleguide that I didn't even know which one to select and `blur`. I thought the native `input` should do the trick – since the event should then bubble up through all the other elements – but it just broke everything... 🤷‍♀